source/concurrent/Promise.ooc

```diff
@@ -118,6 +118,7 @@ Future: abstract class <T> extends _Synchronizer {
 }
 
 _ThreadFuture: class <T> extends Future<T> {
+	_state: _PromiseState
```

Comment:
This should probably not be named `_PromiseState` if it's used for both `Future` and `Promise`. Not related to this PR though and not that important.